docassemble/MAVirtualCourt/data/questions/209A_affidavit.yml

```diff
@@ -0,0 +1,499 @@
+---
+id: Affidavit police included?
+question: |
+  Did you include the police in your description?
+subquestion: |
+   Did you say if the police were involved in this incident?
+yesno: incidents_of_abuse[i].police_included
+---
+id: Affidavit police called?
+question: |
+  Were the police called?
+subquestion: |
+  Were the police involved at all?
+yesno: incidents_of_abuse[i].police_involved
+help:
+  label: Why does this matter?
+  heading: Why does this matter?
+  content: |
+    It may help you give the judge a better picture if you explain how the police were involved. 
+    
+    But you do not have to talk about the police if you do not want to.
+---
+id: Affidavit may want to include police
+question: |
+  You may want to include police in your story.
+subquestion: |
+   It may help you give the judge a better picture if you explain how the police were involved. 
+   
+   You can say how they got involved and what they did. 
+   
+   Do you want to add anything to your story? 
+yesno: incidents_of_abuse[i].add_police
+---
+id: Affidavit did not call police
+question: |
+  If you did not include the police in your story  
+subquestion: |
+   If you needed the police but you did not call them because you were afraid or you could not call them, it may help you to tell the judge. 
+   
+   Do you want to explain why the police were **not** involved?
+yesno: incidents_of_abuse[i].police_why_not
+---
+id: 209A Affidavit Police involvement
+question: |
+  Police involvement
+subquestion: |
+  This is what you have said so far. 
+  
+  % if incidents_of_abuse[i].police_involved:
+    Edit it as much as you need to include how the police **were** involved.
+  % else:
+    Edit it as much as you need to include why the police **were not** involved
+  % endif
+comment: |
+  Here we want to display the incident user has described so far in a fairly large text area and allow the user to edit the text.
+fields:
+  - no label: incidents_of_abuse[i].police_description
+    input type: area
+    default: |
+      ${ incidents_of_abuse[i].description }
+help:
+  label: How do I describe their involvement?
+  heading: How do I describe their involvement?
+  content: |
+    You can put into your affidavit:
+
+    * who called the police, 
+    * did they come in time, 
+    * if they arrested anyone, 
+    * who they arrested,
+    * if ${ defendant } stopped you from trying to call the police,
+    * if the police made a report.
+---
+id: 209A Affidavit you or child injured
+question: |
+  Were you or a child injured?
+subquestion: |
+  Note: 
+  If a child has been abused or witnessed abuse, the judge may: 
+   
+  * appoint a Guardian ad Litem to represent the child's interest. 
```

Comment:
Could this use some plain language work?

Reply:
I say this because we may have someone who can give it a shot.

Reply:
This yml comes from an interview that reads between 1-4th grade reading level and has already been tested with DV survivors.  I think we do not need any more editing for plain language.